### ISSUE TITLE
iptables-restore stall

### ISSUE THREAD

**Issue by kdstich** [Senior-Design-May1601/config#19]
Does the initial setup for Ansible hang up on the IP tables startup task for everyone?  Does it work fine if it is ran again?

**Comment by dborg92** [comment]
update: this issue still persists on my machine with iptable-restore command inside handler.

**Comment by dborg92** [comment]
can ctrl-c out and reprovision to proceed past iptables role.

**Comment by nskinkel** [comment]
Things to investigate:

```
1. What happens if you just load the rules manually over ssh in a completely clean vm (nothing else done, no vagrant)
2. What happens if you load the rules manually over ssh in a vm that's been provisioned exactly up to this role but stopped (no vagrant)
3. What happens if you load the rules manually over ssh *through vagrant* in a clean vm
4. What happens if you load the rules manually over ssh *through vagrant* in a vm that's been provisioned exactly up to this role but stopped
5. What happens if you run the entire playbook on a clean vm?
6. What happens if you run the playbook, ctrl-c to move past, then run the same playbook again?
```

Post the results of these tests.

**Comment by dborg92** [comment]
FINALLY fixed as of commit 3ad63d4 . System always hangs on iptable-restore(I searched for hours and couldn't find any reason why) first time so we use iptables-apply, wait for it to fail, then execute iptables-restore to load the rule-set.
